tests/e2e/e2e_test.go

```diff
@@ -423,65 +430,216 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 			gomega.Ω(err).Should(gomega.BeNil())
 		})
 	})
+	ginkgo.It("can pause a node", func() {
```

Comment:
in diff with avax file we still have bunch of test cases missing, is this ok?

Reply:
these tests were failing and I removed cause they had been also removed from an older avax merge:
see here
https://github.com/chain4travel/camino-network-runner/compare/6d39aae6022bc8a89d3f8cdcea8324a57d05755b...de4c2795e3d1e3443a5bd765413d43523aaa6721#diff-7d1fc1e232b55677582b51e2623aae6d8c15b5f0960ac6100fc7d733cb795dacL149
I asked about the reasoning behind this removal, but the colleagues cannot remember.